app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -188,7 +188,11 @@ public Void call() throws Exception {
                 mActivity.runOnUiThread(new Runnable() {
                     @Override
                     public void run() {
-                        mActivity.getPeopleAdapter().refresh();
+                        try {
+                            mActivity.refreshPeopleDrawer();
```

Comment:
Yes with the help of presence we are sorting (active,inactive,idle) (https://github.com/zulip/zulip-android/blob/master/app/src/main/java/com/zulip/android/models/Person.java#L191) in this PR I have added some more cases to give priority to recent chat one's.(https://github.com/vishwesh3/zulip-android/blob/sort-people-drawer/app/src/main/java/com/zulip/android/models/Person.java#L206)